pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[tool.poetry]
+name = "etl"
+version = "0.1.0"
+description = "Dagster ETL for Wikipedia data."
+authors = ["imlapps"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = ">=3.10,<3.13"
+dagster = "^1.7.9"
+typer = "^0.12.3"
+pydantic = "^2.7.4"
+dagster-webserver = "^1.7.9"
+unidecode = "^1.3.8"
+langchain = "^0.2.5"
+langchain-openai = "^0.1.8"
+pytest-mock = "^3.14.0"
```

Comment:
dev dependency?

Reply:
Yes, I have added this to dev dependencies.